package.json

```diff
@@ -44,6 +48,12 @@
     "karma-mocha": "^0.2.0",
     "karma-mocha-reporter": "^1.1.1",
     "minimist": "^1.2.0",
-    "mocha": "^2.2.5"
+    "mocha": "^2.2.5",
+    "uglifyjs-webpack-plugin": "^1.0.0-beta.3",
+    "webpack": "^3.8.1",
+    "webpack-stream": "^4.0.0"
+  },
+  "dependencies": {
```

Comment:
This `dependencies` key overrides the `dependencies` key above, which contains `json-formatter-js`. You have a duplicate JSON key here.